gen_config/model.conf

```diff
@@ -1 +1 @@
-java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file:///<KILLBILL_REPO>/jaxrs/src/main/java/com/ -o /tmp/output -t PHP_CLIENT_API -p com.ning.billing.jaxrs.json
+java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file://<KILLBILL_REPO>/jaxrs/src/main/java/org -o /tmp/output -t PHP_CLIENT_API -p org.killbill.billing.jaxrs.json
```

Comment:
But this is pointing to the main Killbill repo, no?

The path there is with `org/` (or am i missing something?): https://github.com/killbill/killbill/tree/master/jaxrs/src/main/java/org